### ISSUE TITLE
Bad webm and mp4 recording

### ISSUE THREAD

**Issue by jesusrivas89** [phw/peek#299]
Hi, I have problems with Webm and mp4, especially with the webm. Here are some examples:

Webm at 48 frames:

https://gfycat.com/LittleClutteredIsabellineshrike

Webm with 10 frames:

https://gfycat.com/CommonScentedHammerheadbird

in both videos it looks like it jumps I'm using version 1.3.0 of Peek

now the videos are in another color and when I upload the webm to another hosting they look different

Same Webm to 48 frames in another hosting:

https://webmshare.com/q5MX7

Same webm to 10 frames in another hosting:

https://webmshare.com/oPaYQ

On my Linux Mint Mate 18.3 Sylvia 64Bit looks like this:

https://imgur.com/a/tg1U3

As you can see in Xplayer it looks bad but in VLC and MPV it shows the color correctly

I hope you can help me solve this problem. Thanks in advance

**Comment by phw** [comment]
Do you use Gnome? If so try starting Peek with the ffmpeg backend: `peek -b
ffmpeg`

If that fixes your issue it is the built-in Gnome Shell recording that is
to blame. If not, does it work if you lower the framerate?

-- 
Philipp Wolfer

parolu – work together smoothly
https://www.parolu.io

All-in-one collaboration tool for designers, developers & project managers

jesusrivas89 <notifications@github.com> schrieb am Mo., 26. März 2018,
06:13:

**Comment by jesusrivas89** [comment]
Hi. I do not use Gnome I use Linux Mint Mate but I used the command that em said: peek -b ffmpeg

Open the program perfectly but I get the same result and I lowered the frame rate to 6 and the same result, so I put the two examples of frame rates up to 48 and 10 to show that at both speeds the result is the same

**Comment by phw** [comment]
Yes, unless you actually used gnome there is no difference in starting it with `peek` or `peek -b ffmpeg`, as ffmpeg is the default backend.

I have committed a change that I think will fix your color issues. Not sure about the speedup. I need to look into this.

**Comment by phw** [comment]
@jesusrivas89 Could you test with the just released Peek 1.3.1 ? This should fix your color issues, not sure about the jumping

**Comment by jesusrivas89** [comment]
@phw the color correction is excellent it looks good now, unfortunately the jumps continue to exist. But thank you very much for the new version.
